### ISSUE TITLE
How to override colour variables?

### ISSUE THREAD

**Issue by patrickgordon** [mkhairi/materialize-sass#31]
Hi there

I'm trying to do this: http://materializecss.com/sass.html

How do I overwrite the settings in the _variables.scss when using this gem?

Cheers.

**Comment by alekseychaikovsky** [comment]
first import the colors
```
@import "materialize/components/color"
```

Then override the colors you would like

finally import the framework
```
@import "materialize"
```

**Comment by patrickgordon** [comment]
Thanks @alekseychaikovsky !

**Comment by RalucaP** [comment]
Hello!

I think I did exactly what @alekseychaikovsky said, which is:
```
@import "materialize/components/color";

$blue-grey: (
  "lighten-5":  #a8d0e2
);

$input-border-color: color("blue-grey", "lighten-5") !default;
$input-border: 1px solid $input-border-color !default;

@import "materialize";
```
My color doesn't appear on _color.scss, so I tried to override the closest color to mine, which is the blue-grey, lighten-5.

Although I think I did everything right, it doesn't work, it still uses default blue-grey, lighten-5 color from _color.scss, which is #eceff1

Any idea why my override doesn't work? 
Any help would be really appreciated! :)
